src/three-components/Renderer.ts

```diff
@@ -273,3 +275,5 @@ export class Renderer extends EventDispatcher {
         {type: 'contextlost', sourceEvent: event} as ContextLostEvent);
   }
 }
+
+export let renderer = new Renderer({debug: isDebugMode()});
```

Comment:
Moving `renderer` into here has consequences, but we will only realize them in tests for now. The move means `new Renderer` happens (every) first time that this module gets imported from somewhere, regardless of why it was imported. So, in our test that tests `Renderer`, we will create an extra instance of it and that instance will just sit around and do nothing.

Just putting this out there, since it is bound to come up down the road. No action requested.